Makefile

```diff
@@ -14,25 +20,30 @@ $(DUMMY):
 	$(MAVEN) package --projects $(SERVER)
 
 
+$(RELEASES):
+	@if ! test -d $(RELEASES);then mkdir $(RELEASES) & echo "Created release directory @ $(RELEASES)."; fi;
```

Comment:
There is no need for a conditional here, recipe only commences if target is not found, i.e. This rule will only ever rule if the target folder is missing, so why are we checking if it's missing?

Reply:
fixed [6259890](https://github.com/riecenaidoo/soundboard-gui/pull/47/commits/6259890ab8847b0c9f36c31a95d1ee96449cb5ce)